app/scripts/controllers/incoming-transactions.js

```diff
@@ -56,21 +56,52 @@ class IncomingTransactionsController {
     }, opts.initState)
     this.store = new ObservableStore(initState)
 
+    this.preferencesController.store.subscribe(pairwise((prevState, currState) => {
+      const { featureFlags: { showIncomingTransactions: prevShowIncomingTransactions } } = prevState
```

Comment:
This seems like it'd crash if `featureFlags` was undefined. Likewise for the following line.

Reply:
Also on line 99

Reply:
Fixed